seacrowd/sea_datasets/phost/phost.py

```diff
@@ -170,40 +164,28 @@ def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
         timing = []
         en_sub = []
         vi_sub = []
-        counter = 0
-        
-        if tgt_lang not in ["en", "vi"]:
-            raise NotImplementedError(f"Target language '{tgt_lang}' is not defined.")
-        if src_lang not in ["en", "vi"]:
-            raise NotImplementedError(f"Source language '{src_lang}' is not defined.")
-
         for key, track_id in enumerate(track_ids):
+            with open(os.path.join(filepath["text"], track_id, track_id + ".yaml")) as timing_file:
+                timing = yaml.safe_load(timing_file)
             with open(os.path.join(filepath["text"], track_id, track_id + ".en")) as en_text:
-                    en_sub = [line.rstrip() for line in en_text]
-
-            with open(os.path.join(filepath["text"], track_id, track_id + ".vi")) as vi_text:
-                vi_sub = [line.rstrip() for line in vi_text]
+                en_sub = [line.strip() for line in en_text]
+            with open(
+                os.path.join(filepath["text"], track_id, track_id + ".vi"),
+            ) as vi_text:
+                vi_sub = [line.strip() for line in vi_text]
 
             if self.config.schema == "source":
-                with open(os.path.join(filepath["text"], track_id, track_id + ".yaml")) as timing_file:
-                    timing = yaml.safe_load(timing_file)
-
-                yield key, {
-                    "file": os.path.join(filepath["audio"], track_id + ".wav"), 
-                    "audio": os.path.join(filepath["audio"], track_id + ".wav"), 
-                    "text": en_sub if tgt_lang == "en" else vi_sub, 
-                    "timing": timing
-                }
+                yield key, {"file": os.path.join(filepath["audio"], track_id + ".wav"), "audio": os.path.join(filepath["audio"], track_id + ".wav"), "en_text": en_sub, "vi_text": vi_sub, "timing": timing}
```

Comment:
"en_text" and "vi_text" here is different from the source schema defined in line 83-84

Reply:
Ok. Thanks for noticing that.

Reply:
i noticed sth else.
will fix tonight + upd the output.

Reply:
ok. finished

Reply:
it looks good,
thank you